relay-event-schema/src/protocol/span/compat.rs

```
@@ -95,10 +83,27 @@ impl TryFrom<SpanV2> for CompatSpan {
     }
 }
 
+fn get_string_or_error(attributes: &Attributes, key: &str) -> Option<Annotated<String>> {
+    let annotated = attributes.0.get(key)?;
+    let value: Annotated<Value> = annotated.clone().and_then(|attr| attr.value.value);
+    match value {
+        Annotated(Some(Value::String(description)), meta) => {
+            Some(Annotated(Some(description), meta))
+        }
+        Annotated(None, meta) => Some(Annotated(None, meta)),
+        _ => None,
```

Deliberate silent fail. Assume this is a naming collision, not invalid data.

---

Could add this as a comment to the code, seems useful if someone reads over that code in the future.